apps/frontpage/components/home/hero/manager/index.tsx

```diff
@@ -1,21 +1,21 @@
-import { FC } from "react";
-import { Sidebar } from "./sidebar";
-import { Toolbar } from "./toolbar";
-import { PanelControls } from "./panel-controls";
-import { AnimatePresence } from "framer-motion";
-import { Doc } from "./doc";
-import { PanelInteractions } from "./panel-interactions";
-import { Canvas } from "./canvas";
-import { PanelVisualTesting } from "./panel-visual-testing";
+import { FC } from 'react';
+import { Sidebar } from './sidebar';
+import { Toolbar } from './toolbar';
+import { PanelControls } from './panel-controls';
+import { AnimatePresence } from 'framer-motion';
+import { Doc } from './doc';
+import { PanelInteractions } from './panel-interactions';
+import { Canvas } from './canvas';
+import { PanelVisualTesting } from './panel-visual-testing';
 
 export const Manager: FC<{ slide: number }> = ({ slide }) => {
   return (
     <div
```

Comment:
(Not sure where this feedback belongs.)

I find myself missing the animated cursor. It's pretty hard to parse the difference between each feature, as it is.

The simulated interactions for each feature in the current version also did a lot to illustrate them. E.g. navigating between different stories, changing control values. And while it's not in the current one, I think clicking on a couple different interactions steps would be similarly helpful.

Reply:
I think we should not confuse 2 things:
- A guided tour of the product
- A presentation of the tool in general

With this update I'm going after the second one so people can understand through the tab and the visual the main features. You don't need to go in details but just understand if this looks "cool" enough or with the right keywords to learn more about it.

The guided tour should be done perhaps in a video as you need people to stick to it. The version we have today is way too slow, incomplete and with no way to come back to a specific point.

Reply:
We discussed this one on Slack and in our design Jam and you're right, we could introduce some animations on each slide to make it more interactive. This will require more work but it is worth it. I'll work on it.